src/app/app.component.ts

```diff
@@ -66,7 +66,8 @@ import {ItemComponent} from './dspace/components/item.component';
         { path: "/communities/:id/:page", name: "Communities", component: CommunityComponent },
         { path: "/collections/:id", name: "Collections", component: CollectionComponent },
         { path: "/collections/:id/:page", name: "Collections", component: CollectionComponent },
-        { path: "/items/:id", name: "Items", component: ItemComponent }
+        { path: "/items/:id", name: "Items", component: ItemComponent },
+        { path: "/items/:id/:page", name: "Items", component: ItemComponent }
```

Comment:
Ah I see. 
I have created a branch (not created PR yet) for item views, and I changed that structure a bit as well. Paths are : /items/:id and '/items/full/:id'. 
Have not yet created a PR for that, and we will see then how we can solve the conflict we'll get. 

PR looks good so far! Will check it out further tonight ;-)